mialab/filtering/preprocessing.py

```diff
@@ -29,9 +30,11 @@ def execute(self, image: sitk.Image, params: pymia_fltr.FilterParams = None) ->
         img_arr = sitk.GetArrayFromImage(image)
 
         # todo: normalize the image using numpy
-        warnings.warn('No normalization implemented. Returning unprocessed image.')
+        min_val = img_arr.astype(np.float32).min()
```

Comment:
It's just a side comment. This code should not be in this PR. I mean, here it totally ok, but usually it's convenient to create a new branch from **main**, not from a different branch. When it's form main, then in the PR only updates relevant for this particular PR are visible. :)